File: Saatvik Srivastava/Day 4/Day4A.java
class Solution {
    public int removeDuplicates(int[] arr) {
        int res = 0;
        for(int i =0;i<arr.length;i++){
            if(arr[i] != arr[res]){
                res++;
                arr[res] = arr[i];
            }
        }
        return res+1;
    }
}

File: Akshat Gupta/Day-4A.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        auto nums_end = unique(nums.begin(),nums.end());
        nums.erase(nums_end,nums.end());
        return nums.size();
    }
};
